homeassistant/components/remote_rpi_gpio/switch.py

```diff
@@ -76,12 +76,12 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """Turn the device on."""
-        remote_rpi_gpio.write_output(self._switch, 0 if self._invert_logic else 1)
```

Comment:
Since we don't need `self._invert_logic` anymore it can be removed from that class.